app/retail/templates/vision.html

```diff
@@ -0,0 +1,238 @@
+          <img class="mw-100" src="{% static "v2/images/vision/blockchain.png" %}" alt="twit-roadmap">
+        </div>
+        <div class="media-body">  
+          <h3 class="mb-4">{% trans "Blockchain" %}</h3>
+          <p>{% trans "Will blockchain be in everyone's wallet (Browser), or will it be hidden in the back? (Linux)" %}</p>
+          <p>{% trans "What will Identity on web3 look like?  Will there be a dfb (decentralized facebook)" %}</p>
+          <p>{% trans "What web3 wallet will everyone use?" %}</p>
+          <p>{% trans "What will maturation/consolidation in the blockchain space look like?" %}</p>    
+        </div>
+      </div>
+
+      <div class="media shadow-sm p-5 mb-5 bg-white rounded flex-column flex-md-row animated fadeInRightShort" data-id="2">
+        <div class="w-25 align-self-center text-center mr-md-3 mb-3">
+          <img class="mw-100" src="{% static "v2/images/vision/oss.png" %}" alt="twit-roadmap">
+        </div>
+        <div class="media-body">  
+          <h3 class="mb-4">{% trans "OSS" %}</h3>
+          <p>{% trans "What is a scalable funding model for developers (esp OSS) in a blockchain world?" %}</p>
+          <p>{% trans "Can blockchain solve The Software Paradox?" %}</p>
+          <p>{% trans "Is Github’s centralization a problem?" %}</p>
+        </div>
+      </div>
+
+      <div class="media shadow-sm p-5 mb-5 bg-white rounded flex-column flex-md-row animated fadeInRightShort" data-id="3">
+        <div class="w-25 align-self-center text-center mr-md-3 mb-3">
+          <img class="mw-100" src="{% static "v2/images/vision/future.png" %}" alt="">
+        </div>
+        <div class="media-body">
+          <h3 class="mb-4">{% trans "The Future Of Work" %}</h3>
+          <p>{% trans "Will software developers find work as easily as Uber drivers do today?" %}</p>
+          <p>{% trans "In that scenario, how will ancillary services (Retirement, Insurance, Health Care) build around a ‘future of work’ scenario? I’d imagine, more so, than just making it economically feasible to work 4-5 OSS projects as a full time job, many people wouldn’t want to give up benefits that they accrue in their day-to-day lives." %}</p>
+          <p>{% trans "Are these government based, “employer” based, or none of the above?" %}</p>
+          
+        </div>
+      </div>
+
+      
+
+    </div>
+  </div>
+  <div class="row justify-content-center">
+    <div class="col col-md-4">
+      <div class="text-center mb-5 mx-auto">
+        <p>{% trans "Got input that can inform our vision?" %}</p>
+        <a href="mailto:founders@gitcoin.co" target="_blank" class="btn btn-block btn-gc-purple">{% trans "GET IN TOUCH" %}</a>
+      </div>
+    </div>
+  </div>
+
+  
+  
+</section>
+
+{% include 'shared/footer.html' %}
+{% include 'shared/analytics.html' %}
+{% include 'shared/footer_scripts.html' %}
+{% include 'shared/rollbar.html' %}
+{% include 'shared/messages.html' %}
+</body>
```

Comment:
Could we indent the body block by one as it's nested inside the html tag ?

Reply:
Well seen @thelostone-mc ! 
I was checking the indentation and since I duplicated mission.html to do this page I didn't check that and what is more important there Is not "head" tag in mission.html and vision.html (not a problem with html5 really) 
![image](https://user-images.githubusercontent.com/1358093/42726387-d9bc189e-8769-11e8-9d04-709bcabdff3b.png)
Here is how chrome is rendering the html of mission page in prod and creating the <head> 

I will fix indentation and missing tags for both pages